bitbots_animation_server/src/bitbots_animation_server/animation_node.py

```diff
@@ -167,21 +170,36 @@ def send_animation_request(self):
         self.hcm_publisher.publish(self.anim_msg)
 
     def send_animation(self, first, last, hcm, pose):
+        #HERE
         self.anim_msg.request = False
         self.anim_msg.first = first
         self.anim_msg.last = last
         self.anim_msg.hcm = hcm
         if pose is not None:
+            torque = self.get_torque()
             self.traj_msg.joint_names = []
             self.traj_msg.points = [JointTrajectoryPoint()]
             self.traj_msg.points[0].positions = []
+            self.traj_msg.points[0].effort = []
             for joint in pose:
                 self.traj_msg.joint_names.append(joint)
                 self.traj_msg.points[0].positions.append(pose[joint])
```

Comment:
This looks wrong to me. Should we not use multiple points of the trajectory message for an animation

Reply:
I know you did not touch this, but it might be a good idea to use the message correctly.

Reply:
you mean, that we are currently only using the first point of the message?
this is because we dont want to send a trajectory but a single joint goal. but we could change the message to JointCommand, since we are using this every where else

Reply:
This would require to alter the Animation.msg in humanoid_league_msgs and reduce its functionality. Thus, we keep using it this way, but I've added a comment to clarify that in the code.